regression/example.yaml

```diff
@@ -94,6 +94,7 @@ declarations:
   - decl: namespace sidre
     declarations:
     - decl: class Group
+      imported: True
```

Comment:
This was a bit a kludge to support a class which was actually wrapped by another YAML file. `imported` was meant to indicate that this class was wrapped elsewhere and should not be wrapped again. However, the user had to provide the `fields` section.

Instead I moved the "imported" class into the `typemap` section.  I think that'll work better because it provides a path to using the type.json file which is written by Shroud and is intended to be used to share wrapped classes between YAML files.

Reply:
Thanks for the explanation.  I agree that the new way seems cleaner.